templates/test/ci/patches/apiserver.yaml

```diff
@@ -0,0 +1,10 @@
+kind: KubeadmControlPlane
+apiVersion: controlplane.cluster.x-k8s.io/v1alpha4
```

Comment:
@nilo19 your branch is https://github.com/nilo19/cluster-api-provider-azure/tree/chore/mix not rebased (297 commits behind main) this should be v1beta1. Try rebasing and aplying the changes on the newer templates.

Reply:
Thanks for the info.